plugins/action_update_review_status/src/updateReviews.ts

```diff
@@ -10,14 +11,16 @@ interface Review {
   reviewStatus: ReviewStatus
 }
 
-async function updateReviews(input: any, DBConnect: any) {
+async function updateReviews({ parameters, applicationData, DBConnect }: ActionPluginInput) {
   const db = databaseMethods(DBConnect)
 
   console.log('Updating reviews status...')
 
-  const { applicationId, applicationData, changedApplicationResponses = [] } = input
-  const stageId = input?.stageId || applicationData.stageId
-  const level = input?.level || applicationData?.reviewData?.level || 1
+  const applicationId = parameters?.applicationId ?? applicationData?.applicationId
+
+  const { changedApplicationResponses = [] } = parameters
```

Comment:
Should also consider it can be coming from `applicationData` since it's optional

Reply:
We can't get this from applicationData -- it's not the main "responses" object, it's specifically the *changed* responses, which are generated by the output of `trimResponses` action.

Reply:
Hmmm. Cool, I think then the docs needs updates: `changedApplicationResponses` as required field

Reply:
Strictly speaking, it's not required, since it defaults to an empty array: ` const { changedApplicationResponses = [] } = parameters`

And it is valid to pass nothing in, the plugin will run, it'll just come back with "Nothing changed".